mlir/lib/Conversion/LLVMCommon/Pattern.cpp

```diff
@@ -381,3 +381,95 @@ LogicalResult LLVM::detail::oneToOneRewrite(
   rewriter.replaceOp(op, results);
   return success();
 }
+
+static unsigned getBitWidth(Type type) {
+  if (type.isIntOrFloat())
+    return type.getIntOrFloatBitWidth();
+
+  auto vec = cast<VectorType>(type);
```

Comment:
I think we need to assert this is not a scalable vector or bail out in some other way

Reply:
Good catch, updated verifier to reject scalable vectors, added assert here.